index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"/>
+    <link rel="stylesheet" href="style.css">
+    <title>Leo's Portfolio</title>
+</head>
+<body>
+    <nav>
+        <div><a href="index.html">Leo</a></div>
+        <div><a href="#"><i class="fa-solid fa-bars"></i></a></div>
+    </nav>
+    <div class="nav-items">
+        <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#"><i class="fa-regular fa-envelope"></i></a></li>
+        </ul>
+    </div>
+    <!--Home Section-->
+    <div class="home">
+        <div class="intro">
+            <span>Hey There. I'm Leo</span>
+        </div>
+        <div class="job">
+            <span>I'am a full-stack Web developer</span>
+        </div>
```

Comment:
- [ ] Dear @popey17, This text should be wrapped inside one of a `h2 - h6` tags. This way it will be semantically correct and the screen readers and other technologies will thank you.